src/components/ui/HowItWorks.tsx

```diff
@@ -0,0 +1,33 @@
+import {cn} from '@/lib/utils/client/tailwind'
+
+type HowItWorksProps = {
+  items: {title: string; description: JSX.Element | string}[]
+  header?: string
+  color?: string
```

Comment:
Btw this could be just a `className` as really it can be used for anything and `bg-grey-90` could be just part of the default styles.